Implementation/Default-Routing/Execute-Request/index.md

```diff
@@ -30,12 +30,12 @@ Working with dynamics is simpler in some cases especially with regards to refere
 
 	@CurrentPage.markDown
 
-However, dynamics do not provider any intellisense and are compiled at runtime
+However, dynamics does not provide any intellisense and is compiled at runtime.
```

Comment:
This sounds a bit wrong to me, is the word `dynamics` not plural? So `do` and `are` are the correct words?

Reply:
@jmayntzhusen it's confusing because 'dynamics' can be considered to be the 'subject' of working with dynamic objects, but I think in this context it would be clearer still to have:

However, a _dynamic_ object does not provide any intellisense and is compiled at runtime.

Reply:
Yep @marcemarc 's suggestion should clear it up, could you make that change @abbas10m ? 🙂

Reply:
Updated. Thanks for the feedback @jmayntzhusen and @marcemarc  :)